src/ui/qgsoptionsbase.ui

```diff
@@ -2005,7 +1951,7 @@
                   <item>
                    <widget class="QgsCollapsibleGroupBox" name="mSimplifyDrawingGroupBox">
                     <property name="title">
-                     <string>Enable feature si&amp;mplification by default for newly added layers</string>
+                     <string>Enable fea&amp;ture simplification by default for newly added layers</string>
```

Comment:
Are these shortcut changes deliberate?

Reply:
Absolutely not, my qt-designer made something strange here...

Reply:
As soon as I open/save this file with my Qt-Designer, shortcuts are changed... Anyway, as *trustProject* is not a global option anymore, I reverted changes on *qgsoptionsbase.ui *.